client/src/constants/archiveParticipantsConstants.js

```diff
@@ -1,4 +1,4 @@
-export const FailedCohortReason = 'Failed cohort, does not wish to continue in HCAP';
+export const FailedCohortReason = 'Unsuccessful cohort, does not wish to continue in HCAP';
```

Comment:
Do we want to change the related constants to match the wording choice here?

Reply:
Done